plug.py

```diff
@@ -66,19 +66,23 @@ def __init__(self, name: str, loadfile: str | None, plugin_logger: logging.Logge
             try:
                 filename = 'plugin_'
                 filename += name.encode(encoding='ascii', errors='replace').decode('utf-8').replace('.', '_')
-                module = importlib.machinery.SourceFileLoader(
-                    filename,
-                    loadfile
-                ).load_module()
-                if getattr(module, 'plugin_start3', None):
-                    newname = module.plugin_start3(os.path.dirname(loadfile))
-                    self.name = str(newname) if newname else self.name
-                    self.module = module
-                elif getattr(module, 'plugin_start', None):
-                    logger.warning(f'plugin {name} needs migrating\n')
-                    PLUGINS_not_py3.append(self)
+                spec = importlib.util.spec_from_file_location(filename, loadfile)
+
+                if spec is not None and spec.loader is not None:  # Check if spec and spec.loader are not None
```

Comment:
This comment literally repeats exactly what the code states.  I can't imagine any replacement would be any less obvious, so just remove it ?